CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 6.0.2
+  - Update of GeoLite2 DB
```

Comment:
Would you mind adding a link to this issue, as shown below
```suggestion
  - Update of GeoLite2 DB  [#157](https://github.com/logstash-plugins/logstash-filter-geoip/pull/157)
```

Reply:
Added the suggestion